pkg/module/go_get_fetcher.go

```diff
@@ -105,8 +105,7 @@ func downloadModule(goBinaryName string, envVars []string, fs afero.Fs, gopath,
 	const op errors.Op = "module.downloadModule"
 	uri := strings.TrimSuffix(module, "/")
 	fullURI := fmt.Sprintf("%s@%s", uri, version)
-
-	cmd := exec.Command(goBinaryName, "mod", "download", "-json", fullURI)
+	cmd := cmd(goBinaryName, "mod", "download", "-json", fullURI)
```

Comment:
While it technically _works_, I would argue against shadowing the new package-level `cmd` variable like this.  Either rename the package variable or use a different local variable.